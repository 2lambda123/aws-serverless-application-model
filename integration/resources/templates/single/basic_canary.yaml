Resources:
  SyntheticsCanary:
    Type: AWS::Serverless::Canary
    Properties:
      Handler: pageLoadBlueprint.handler
      InlineCode: |
        def handler(event, context):
          print("Hello, world!")
      Runtime: syn-python-selenium-1.0
      StartCanaryAfterCreation: true
      Schedule:
        Expression: rate(0 minute)
