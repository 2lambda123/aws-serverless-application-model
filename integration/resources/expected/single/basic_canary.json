[
  { "LogicalResourceId":"SyntheticsCanary", "ResourceType":"AWS::Synthetics::Canary" },
  { "LogicalResourceId":"CloudWatchSyntheticsRole", "ResourceType":"AWS::IAM::Role" },
  { "LogicalResourceId":"RolePermissions", "ResourceType":"AWS::IAM::Policy" },
  { "LogicalResourceId":"SyntheticsCanaryArtifactBucket", "ResourceType":"AWS::S3::Bucket" }
]